### PR TITLE
Fix some shared alert ownership

### PR DESCRIPTION
**Pull request by QuentinBisson**
Before adding a new alerting rule into this repository you should consider creating an SLO rules instead.
SLO helps you both increase the quality of your monitoring and reduce the alert noise.

* How to create a SLO rule: https://github.com/giantswarm/sloth-rules#how-to-create-a-slo
* Documentation: https://intranet.giantswarm.io/docs/monitoring/slo-alerting/

---
Towards: https://github.com/giantswarm/...

This PR sets ownership to turtles for shared alerts between providers but still pages the cluster provider team

### Checklist

- [ ] Update CHANGELOG.md
- [ ] Add [Unit tests](https://github.com/giantswarm/prometheus-rules/#testing)
- [ ] Follow [Alert structure](https://github.com/giantswarm/prometheus-rules/#how-alerts-are-structured)
- [ ] Consider [creating a dashboard](https://docs.giantswarm.io/getting-started/observability/grafana/custom-dashboards/) ([guidelines](https://intranet.giantswarm.io/docs/product/ux/guidelines/dashboards/)) (if it does not exist already) to help oncallers monitor the status of the issue.
- [ ] Request review from oncall area, as well as team (e.g: `oncall-kaas-cloud` GitHub group).

### CODE REVIEW

File: helm/prometheus-rules/templates/platform/honeybadger/alerting-rules/chart.rules.yml [hunk 1]
```diff
@@ -10,6 +10,25 @@ spec:
   groups:
   - name: chart
     rules:
+    - alert: ChartOperatorDown
```
Comment (QuentinBisson) [inline helm/prometheus-rules/templates/platform/honeybadger/alerting-rules/chart.rules.yml:13]: Moved from shared `up.rules.yml`

File: helm/prometheus-rules/templates/kaas/turtles/alerting-rules/kubelet.rules.yml [hunk 2]
```diff
@@ -8,6 +8,25 @@ metadata:
   namespace: {{ .Values.namespace  }}
 spec:
   groups:
+  - name: cadvisor
```
Comment (QuentinBisson) [inline helm/prometheus-rules/templates/kaas/turtles/alerting-rules/kubelet.rules.yml:11]: Taken from the up.rules.yml

File: helm/prometheus-rules/templates/shared/alerting-rules/certificate.all.rules.yml [hunk 3]
```diff
@@ -36,17 +36,6 @@ spec:
         severity: page
         team: phoenix
         topic: cert-manager
-    - alert: KyvernoCertificateSecretWillExpireInLessThanTwoDays
```
Comment (QuentinBisson) [inline helm/prometheus-rules/templates/shared/alerting-rules/certificate.all.rules.yml:39]: Moved to kyverno.rules

**Review by Gacko — APPROVED** [review]

File: helm/prometheus-rules/templates/kaas/turtles/alerting-rules/kubelet.rules.yml [hunk 4]
```diff
@@ -8,6 +8,25 @@ metadata:
...
+    rules:
+      - alert: CadvisorDown
+        annotations:
+          description: '{{`Cadvisor ({{ $labels.instance }}) is down.`}}'
+          opsrecipe: kubelet-is-down/
+        expr: label_replace(up{app="cadvisor"}, "ip", "$1", "instance", "(.+):\\d+") == 0
```
Comment (QuentinBisson) [inline helm/prometheus-rules/templates/kaas/turtles/alerting-rules/kubelet.rules.yml:17]:
```suggestion
        expr: label_replace(up{job="kubelet", metrics_path="/metrics/cadvisor"}, "ip", "$1", "instance", "(.+):\\d+") == 0
```

**Pull request merged**